mrp_bom_purchase/tests/test_mrp_bom_purchase.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestMrpBomPurchase(TransactionCase):
+    def setUp(self):
+        super(TestMrpBomPurchase, self).setUp()
```

Comment:
```suggestion
        super().setUp()
```

Cette syntaxe devrait résoudre le soucis UP008...

Reply:
haha, ah oui jsuis con j'ai lu le truc à l'envers